test/Game-test.js

```diff
@@ -0,0 +1,78 @@
+const chai = require('chai');
+const expect = chai.expect;
+const Game = require('../js/Game.js');
+const spies = require('chai-spies');
+chai.use(spies);
+global.data = require('../js/data.js');
+global.domUpdates = require('../js/DOM.js');
+global.Round = require('../js/Round.js');
+// global.index = require('../js/index.js');
```

Comment:
You shouldn't need to import your `index.js` file here, you can get rid of this

Reply:
With require statements, a general convention is to require 3rd party libraries first (code you didn't write), then leave a blank line, then require any of your own dependencies (like Game.js). I'd re-organize this to something like:

```js
const chai = require('chai');
const expect = chai.expect;
const spies = require('chai-spies');
chai.use(spies);

const Game = require('../js/Game.js');
global.Round = require('../js/Round.js');
global.data = require('../js/data.js');
global.domUpdates = require('../js/DOM.js');
```